src/golem_network/golem_network.ts

```diff
@@ -1,12 +1,38 @@
-import { TaskExecutor } from "../executor";
+import { TaskExecutor, YagnaOptions } from "../executor";
+import { JobStorage } from "../job";
 import { PackageOptions } from "../package";
 import { Worker } from "../task";
 
 export interface GolemNetworkConfig {
```

Comment:
maybe it's not for this time, but in all old entities we use the `xxxOptions` convention to name interfaces, so maybe it would be better to use `GolemNetworkOptions` for consistency..?